vllm/model_executor/models/minicpmv.py

```diff
@@ -75,10 +76,10 @@ def get_abs_pos(abs_pos, tgt_size):
 
 
 # https://github.com/facebookresearch/mae/blob/efb2a8062c206524e35e47d04501ed4f544c0ae8/util/pos_embed.py#L20
-def get_2d_sincos_pos_embed(embed_dim,
-                            grid_size,
-                            cls_token=False,
-                            version=2.0):
+def get_2d_sincos_pos_embed(embed_dim: int,
+                            grid_size: Union[int, Tuple[int, int]],
+                            cls_token: bool = False,
+                            version: float = 2.0):
```

Comment:
To avoid floating-point errors, can we make the version a tuple instead (e.g. `(2, 0)` for v2, `(2, 5)` for v2.5)?

Reply:
Could you provide some specific examples? I don't see the harm.

Reply:
I just mean that it is easy to accidentally perform mathematical operations on the version, which doesn't really make sense.

Reply:
emmmm, alright, I'll handle this. Does it need to be changed to a `Tuple[int, int]` type?

Reply:
Yes.